lib/org/openpsa/user/locale/default.en.txt

```diff
@@ -294,6 +294,10 @@ Person %s created
 Person %s saved
 ---STRINGEND
 
+---STRING password reset and mail sent
+Password has been reset and sent to the user
```

Comment:
I wonder if if would be nicer to print the recipient email address instead of just saying "the user"

Reply:
Changed